README.md

```diff
@@ -152,6 +152,33 @@ with NodeNetworkConfigurationPolicy(
     yield nncp
 ```
 
+### Adding Role / ClusterRole
+Add multiple rules to a specific Role or ClusterRole:
+```python
+latency_role = Role(
+    name="latency-role",
+    namespace=checkup_ns.name,
+)
+latency_role.add_rule(
+    api_groups=["kubevirt.io"],
+    permissions_to_resources=["virtualmachineinstances"],
+    verbs=["get", "create", "delete"],
+)
+latency_role.add_rule(
+    api_groups=["subresources.kubevirt.io"],
+    permissions_to_resources=["virtualmachineinstances/console"],
+    verbs=["get"],
+)
+latency_role.add_rule(
+    api_groups=["k8s.cni.cncf.io"],
+    permissions_to_resources=["network-attachment-definitions"],
+    verbs=["get"],
+)
```

Comment:
Should have `add_rules` which gets a list of dicts with rules.
 `add_rules` should call  `add_rule` which should add one rule at a time.
The idea is to be able to create a resource with multiple rules (should be updated in the code in all places)

Reply:
Ack.